features/FEATURE_LWIP/TESTS/mbedmicro-net/udp_echo_parallel/main.cpp

```diff
@@ -23,6 +23,8 @@
 #define MBED_CFG_UDP_CLIENT_ECHO_THREADS 3
 #endif
 
+#define PARALLEL_STACK_SIZE 1024
```

Comment:
Nit: This should have an ifdef gaurd to allow users to override the setting from compilation. For the other test setting I adopted the naming convention above. I'm not sure this is right, but was in use in other parts of the test. (These aren't exposed to mbed config but _does_ allow it to be easily in the future).